CHANGELOG.md

```diff
@@ -10,12 +10,16 @@ Note that Sockeye has checks in place to not translate with an old model that wa
 
 Each version section may have have subsections for: _Added_, _Changed_, _Removed_, _Deprecated_, and _Fixed_.
 
+## [1.15.0]
```

Comment:
why the version bump? old models should still be compatible, just faster, no?

Reply:
I did it because the signature of `--bucket-width` for translate.py has changed (it now only accepts a single number). 
Old models are still compatible though.
What do you think?

Reply:
I personally would like to avoid forcing people to having to retrain as much as possible and would like the major version at 1.14, but I see the point regarding the CLI change.